terraform/variables.tf

```diff
@@ -232,6 +232,22 @@ variable "gcs_source" {
   }
 }
 
+variable "install_workload_agent" {
+  description = "Whether to install workload-agent on the database VM."
+  type        = bool
+  default     = true
+}
+
+variable "oracle_metrics_secret" {
+  description = "Fully qualified name of the Secret Manager secret that stores the Oracle database user's password. This user is specifically configured for the workload-agent to enable metric collection."
+  type        = string
```

Comment:
Unless we require this parameter to be specified, we need a default, even if it's null or an empty string (and also validation that will succeed in this case)